spring-cloud-stream-test-support/src/test/java/org/springframework/cloud/stream/test/example/ExampleTest.java

```diff
@@ -39,8 +39,9 @@
  * correctly.
  */
 @RunWith(SpringJUnit4ClassRunner.class)
-@SpringBootTest(classes = ExampleTest.MyProcessor.class, properties = { "server.port=-1", "--spring.cloud.stream.bindings.input.contentType=text/plain", "--spring.cloud.stream.bindings.output.contentType=text/plain",
-		"--spring.main.allow-bean-definition-overriding=true"})
+@SpringBootTest(classes = ExampleTest.MyProcessor.class, properties = { "server.port=-1",
```

Comment:
The `server.port=-1` has to be replaced with the `webEnvironment = SpringBootTest.WebEnvironment.NONE`